stable/aws-s3-proxy/templates/deployment.yaml

```diff
@@ -8,13 +8,11 @@ spec:
   replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
-      app.kubernetes.io/name: {{ include "aws-s3-proxy.name" . }}
-      app.kubernetes.io/instance: {{ .Release.Name }}
+{{ include "aws-s3-proxy.labels" . | indent 6 }}
```

Comment:
I don't think this part should change as selector labels are immutable and shouldn't be treated like the other labels 🙂

Reply:
Hi there, thanks for your comment!

Are you suggesting that the changes to `matchLabels` might cause breaking changes? If that’s the case, I’ll revert the changes to `matchLabels`. 🙏

Reply:
Exactly, they are immutable, you can't change them without deleting and recreating the resource. So I would just not change this part.

Reply:
I’ve reverted it. Thanks for pointing it out and for your kind review! 😄